<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1556015816462" stop="1556015834955" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default Suite : afbtest</name>
    <title>Default Suite : afbtest</title>
    <test-cases>
        <test-case start="1556015816987" stop="1556015834926" status="failed">
            <name>checkHeaderCustomersSayButtonNavigateToCustomersSayBlock</name>
            <failure>
                <message>AssertionError: expected [892] but found [5266]</message>
                <stack-trace>java.lang.AssertionError: expected [892] but found [5266]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at FinanceTest.checkHeaderCustomersSayButtonNavigateToCustomersSayBlock(FinanceTest.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:127)
</stack-trace>
            </failure>
            <steps>
                <step start="1556015831989" stop="1556015834879" status="passed">
                    <name>clickOnHeaderCustomersSay</name>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default Suite"/>
                <label name="testGroup" value="afbtest"/>
                <label name="testClass" value="FinanceTest"/>
                <label name="testMethod" value="checkHeaderCustomersSayButtonNavigateToCustomersSayBlock"/>
                <label name="host" value="Nazar"/>
                <label name="thread" value="6660@Nazar.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
